Plandomizer.py

```diff
@@ -921,7 +921,12 @@ def configure_gossip(self, spoiler, stoneIDs):
             matcher = self.pattern_matcher(name)
             stoneID = pull_random_element([stoneIDs], lambda id: matcher(gossipLocations[id].name))
             if stoneID is None:
-                raise RuntimeError('Gossip stone unknown or already assigned in world %d: %r. %s' % (self.id + 1, name, build_close_match(name, 'stone')))
+                # Allow planning of explicit textids
+                match = re.match('^\$([a-fA-F0-9]{4})$', name)
```

Comment:
Why `$` rather than `0x` as prefix? The gossip stone IDs are listed with `0x` in Hints.py

Reply:
I wrote this code so long ago (it just took months before I got around to PRing it), so I'm not entirely certain why I chose $ as a prefix at the time.  It was done to support a plando I was working on.  It was also consistent with another PR I submitted that used ${one or more hex bytes} for inserting arbitrary control codes in text.

Considering I don't think we'll ever have a hintstone name that's 4 hexadecimal digits by itself, the requirement here can probably be more lenient -- i.e.

`$:i^(?:\$|x|0x)?([0-9a-f]{4})$` in case-insensitive mode.